test-cases/features/dns-cluster-5min.yaml

```diff
@@ -1,6 +1,6 @@
 # TODO: need to qualify
 
-test_duration: 20
+test_duration: 77
```

Comment:
Place this and another timeout change to a separate commit because both are unrelated to the main goal of the main change here.